src/templates/default/proposals/_proposal_update.html

```diff
@@ -26,11 +26,15 @@ <h3 class="dashboard-header">
 				{% trans 'Proposal' %}
 			</a>
 		</li>
-		<li role="presentation">
-			<a href="#reviews" aria-controls="reviews" role="tab" data-toggle="tab">
-				{% trans 'Reviews' %}
-			</a>
-		</li>
+		{% get_model_verbose_name_raw object as obj_verbose_name %}
+		{% if 'tutorial proposal' != obj_verbose_name %}
```

Comment:
nit: change the sequence for readability

```suggestion
		{% if obj_verbose_name != 'tutorial proposal' %}
```